packages/ui/src/components/editor/panels/Viewport/tools/GizmoTool.tsx

```diff
@@ -44,66 +44,125 @@ function Placer() {
   )
 }
 
-export default function GizmoTool() {
+export default function GizmoTool({
```

Comment:
for some reason, i was unable to get the gizmo by clicking on the model. it used to appear just by clicking on the model. 

**dev branch**

https://github.com/user-attachments/assets/673b9183-0692-4bf7-a0db-13d502b1d1c1

**this branch**

https://github.com/user-attachments/assets/09f7870d-6865-43e5-995a-555da04d571a

Reply:
good catch, thanks